README.md

```diff
@@ -1,2 +1,6 @@
-# pydsa-viz-examples
-PyDSA Visualization Examples
+
```

Comment:
Wrong file changed.

Your README should be inside your folder.

Don't change the repo's README please.